### PR TITLE
Event Registration

### PR DESCRIPTION
**Pull request by hilmarf**
let's keep track of the branch...

### CODE REVIEW

**Review by roesslerj — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -48,7 +48,7 @@
     <sonar.projectKey>${sonar.organization}_cwa-${project.artifactId}</sonar.projectKey>
 
     <!-- https://spring.io/projects/spring-boot -->
-    <spring-boot.version>2.4.4</spring-boot.version>
+    <spring-boot.version>2.4.3</spring-boot.version>
```
Comment (ioangut) [inline pom.xml:51]: We change the version of spring boot to 2.4.4. Maybe we should keep it.

Comment (hilmarf) [reply pom.xml:51]: @ioangut - right. Main branch versions are running fine, correct? Then please keep the newer version

Comment (FelixRottler) [reply pom.xml:51]: it should be the same for spring cloud version then @ioangut

**Review by ioangut — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (ioangut) [reply pom.xml]: I checked and Spring Cloud version is the same @FelixRottler

**Pull request merged**